conda_index/index/__init__.py

```diff
@@ -309,152 +313,6 @@ def _make_channeldata_index_html(channel_name, channeldata):
     return rendered_html
 
 
-def _make_rss(channel_name, channeldata):
-    return rss.get_rss(channel_name, channeldata)
-
-
```

Comment:
Anticipating a "no conda dependency" or "optional current_repodata" feature, we move these to another module.